client/reader/start/style.scss

```diff
@@ -226,12 +219,16 @@
 	font-family: $serif;
 	font-size: 16px;
 	font-weight: 600;
-	line-height: 1;
+	line-height: 1.3;
 	margin-bottom: 5px;
 
 	&:hover {
 		color: darken( $gray, 10% );
 	}
+
+	&:visited {
+		color: $gray-dark;
```

Comment:
I think we could do `color: inherit` here instead of duplicating the value from above. That way we only have to change one value in the future if needed.